modules/appeals_api/spec/models/notice_of_disagreement_spec.rb

```diff
@@ -86,6 +86,17 @@
     end
   end
 
+  context 'birth date isn\'t in the past' do
```

Comment:
Let's update this to 'Veteran birth date...' since we now have claimant as well :)